src/app/clusters/scenes/scenes.cpp

```diff
@@ -44,32 +44,22 @@ uint8_t emberAfPluginScenesServerEntriesInUse = 0;
 EmberAfSceneTableEntry emberAfPluginScenesServerSceneTable[MATTER_SCENES_TABLE_SIZE];
 #endif
 
-static bool readServerAttribute(EndpointId endpoint, ClusterId clusterId, AttributeId attributeId, const char * name,
-                                uint8_t * data, uint8_t size)
+static bool logReadError(EmberAfStatus status, const char * attributeName)
 {
-    bool success = false;
-    if (emberAfContainsServer(endpoint, clusterId))
+    // Don't log errors for the "this cluster is not even supported" cases.
+    if (status != EMBER_ZCL_STATUS_SUCCESS && status != EMBER_ZCL_STATUS_UNSUPPORTED_ENDPOINT &&
+        status != EMBER_ZCL_STATUS_UNSUPPORTED_CLUSTER)
```

Comment:
```suggestion
        status != EMBER_ZCL_STATUS_UNSUPPORTED_CLUSTER)
```

I guess your next PR will be "Stop using non-namespaced status code ids in core SDK code" :)

Reply:
Yes, I would like to do that too....   ;)